experiment/build/gcb_build.py

```diff
@@ -39,84 +40,76 @@
 logger = logs.Logger('builder')  # pylint: disable=invalid-name
 
 
+def _get_buildable_images(fuzzer=None, benchmark=None):
+    return docker_images.get_images_to_build([fuzzer], [benchmark])
+
+
 def build_base_images():
     """Build base images on GCB."""
-    _build(get_build_config_file('base-images.yaml'), 'base-images')
+    image_templates = {'base-image': _get_buildable_images()['base-image']}
```

Comment:
Would `image_templates = _get_buildable_images()` work?

Reply:
No, it will return other images apart from base-image as well.